pkg/kopia/cli/repository/repository_connect.go

```diff
@@ -54,6 +54,6 @@ func Connect(args ConnectArgs) (*safecli.Builder, error) {
 			args.RepoPathPrefix,
 			args.Logger,
 		),
-		optPointInTime(args.PointInTime),
+		optPointInTime(args.Location, args.PointInTime),
```

Comment:
`point-in-time` flag is present in other repository commands for `s3` as well. Should it just be a part of optStorage?

Reply:
I think we should keep it as is at least for now. Kopia [doesn't support the `point-in-time`](https://github.com/kopia/kopia/blob/aa5e4cfb3326c999912afd5140740bf59e5058a4/cli/storage_s3.go#L50) option during `kopia repository create` for S3 and Azure.

```go
	if isCreate && c.s3options.PointInTime != nil && !c.s3options.PointInTime.IsZero() {
		return nil, errors.New("Cannot specify a 'point-in-time' option when creating a repository.")
	}
	// Similar code for Azure
```

Also, we do not use the `kopia repository repair` and `kopia repository sync` commands. 

If we still wish to move the `point-in-time` option into `optStorage`, then we must add an extra argument to `optStorage(..., isCreated bool)` and pass it on to the providers as `<provider>.New(..., isCreated bool)`. This would mandate moving command logic into the storage provider.